### PR TITLE
Series Operation

### LINKED ISSUE #3520: Improve data frame serialize/deserialize
The current data frames I don't believe have full serialize/deserialize, which we'll need for plugin support.

Here's a sample of implementing it for a couple structs with columns: https://gist.github.com/jonathandturner/74e8a471922c3129c6019f7b732c5a62

cc @elferherrera

### PR DESCRIPTION
**Pull request by elferherrera**
This PR introduces the Series type for nushell. A series represent a column from a frame. It can be used to do columnar operations or to create masks that can be used later to filter a dataframe. 

The series object allows the next workflow:

image

Now we can also create series from a list of values and they can be added to a dataframe

image

@jonathandturner The new version of polars now implements a Serde implemetation. Considering that, this new PR could closes #3520. What do you think?

### CODE REVIEW

File: crates/nu-command/src/commands/dataframe/drop_duplicates.rs [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+    fn name(&self) -> &str {
+        "pls drop_duplicates"
+    }
+
+    fn usage(&self) -> &str {
+        "Drops duplicate values in dataframe"
+    }
+
+    fn signature(&self) -> Signature {
+        Signature::build("pls drop_duplicates")
```
Comment (jonathandturner) [inline crates/nu-command/src/commands/dataframe/drop_duplicates.rs:20]: Same (there are a few other places, but I'll just mention these and you can update the others)

**Review by jonathandturner — COMMENTED** [review]
This looks really good. The cleanups you did for the commands make it easier to follow.

I did have one nit: the names we use in Nushell should be `do-this` rather than `do_this`. There are a couple places we use the second form but we're trying to fix them so everything is in the first form. We can land this and then fix them later, or go ahead and fix them now before we land.

File: crates/nu-command/src/commands/dataframe/drop.rs [hunk 2]
```diff
@@ -34,7 +31,7 @@ impl WholeStreamCommand for DataFrame {
     fn examples(&self) -> Vec<Example> {
         vec![Example {
             description: "drop column a",
-            example: "[[a b]; [1 2] [3 4]] | pls convert | pls drop [a]",
+            example: "[[a b]; [1 2] [3 4]] | pls to_df | pls drop [a]",
```
[the remark this answers is not in the document]
Comment (elferherrera) [reply crates/nu-command/src/commands/dataframe/drop.rs:34]: @jonathandturner It makes sense to add those changes now to set the foundation for the next to come. I'll push those changes now

**Comment by jonathandturner** [comment]
Looks good!

**Pull request merged**